### PR TITLE
add NextTrigger timestamp variable

### PR DESCRIPTION
**Pull request by KingKahn123**
I have added a new variable which indicates when the next light must be switched on or off. As I still have the problem that in some cases the internal timer gets inactive, I would use this "NextTrigger" variable as a WatchDog.

### CODE REVIEW

**Review by paresy — CHANGES_REQUESTED** [review]
Hi! And sorry for taking so long to review this. I just haven't seem it in my list. 

Then change looks good and small and i'd love to merge it. Just a few points:
a) I'd prefer to use an Integer Variable with the UnixTimestampDate profile (This also solves the issue, that you did not translate 'Keiner')
b) We need all checks to be "green" :)

**Comment by KingKahn123** [comment]
Hi, now I have changed it and everything is green. Is it ok with you?

File: AnwesenheitsSimulation/module.php [hunk 1]
```diff
@@ -25,6 +25,7 @@ public function Create()
 
         //Variables
         $this->RegisterVariableString('SimulationView', $this->Translate('Simulation preview'), '~HTMLBox');
+        $this->RegisterVariableInteger('NextTrigger', $this->Translate('Next Trigger'), '');
```
Comment (paresy) [inline AnwesenheitsSimulation/module.php:28]: The last parameter (profile) should be ~UnixTimestamp. (See here: https://www.symcon.de/en/service/documentation/components/visualizations/object-presentation/datetime/)

File: AnwesenheitsSimulation/module.php [hunk 2]
```diff
@@ -376,8 +378,10 @@ public function UpdateTargets()
 
         if (isset($NextSimulationData['nextSwitchTimestamp'])) {
             $this->SetTimerInterval('UpdateTargetsTimer', ($NextSimulationData['nextSwitchTimestamp'] - $this->getTime() + 1) * 1000);
+            $this->SetValue('NextTrigger', ($NextSimulationData['nextSwitchTimestamp'] - $this->getTime() + 1) * 1000);
```
Comment (paresy) [inline AnwesenheitsSimulation/module.php:381]: I think this should be $NextSimulationData['nextSwitchTimestamp'] only. Can you verify this? (Your current code shouldn't work properly!?)

**Review by paresy — CHANGES_REQUESTED** [review]
I added two comment. Could you have a look?

**Comment by paresy** [comment]
@TillBrede: Sobald er die Änderungen eingebaut hat - magst du das mal auschecken und Funktional überprüfen? :-)

**Comment by paresy** [comment]
I will close this for now. Please reopen, if you would like to pick up adding the required changes :-)

**Pull request closed without merging**